utilities/src/with_std/health.rs

```diff
@@ -6,10 +6,29 @@
 use std::{net::IpAddr, sync::Arc};
 
 use crate::{task_scope, Port};
+use clap::Args;
 use serde::Deserialize;
 use tracing::info;
 use warp::Filter;
 
+#[derive(Args, Debug, Clone, Default)]
+pub struct HealthCheckOptions {
+	#[clap(
+		id = "HEALTH_CHECK_HOSTNAME",
+		long = "health_check.hostname",
+		help = "Hostname for this server's healthcheck. Requires HEALTH_CHECK_PORT to be set as well.",
```

Comment:
`health_check.port` instead of HEALTH_CHECK_PORT - since the user won't see this id - they need the id they can use, same below for hostname